### DISCUSSION TITLE
Kann man aus einem anonym abgespeicherten Projekt den Wiederherstellungscode rekonstruieren?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#181] [Q&A]
Wir haben eine Anfrage von einer Agrargemeinschaft, welche im WebGIS Projekte angelegt und den Wiederherstellungscode verloren hat. Kann man aus dem Verzeichnisnamen zB: _anonym$f8fd266396474005872a37e69b1bcd5f den Wiederherstellungscode rekonstruieren?

**Comment by gis-eni** [comment]
Nein das ist per Definition nicht möglich. 
Aber hast du das richtige Verzeichnis gefunden?

Wenn ja:

die sollten ein neues Projekt unter einem neuen Wiederherstellungscode anlegen. Das das Projekt sollten sie einen Redlining Text, zb: "bitte, bitte, bitte wieder alles herstellen" anlegen. Den neuen Ordner kannst du dann suchen (über den Text).
Dann kannst du probieren, die Files aus den alten Ordner in den neuen zu kopieren. Wenn du Glück hast, sehen sie die Files wieder.

Grundsätzlich: Wenn diese Projekte reine "Redlining" Projekte sind, also es eigentlich ums Redlining geht, sollte man den Anwendern beibringen, dass sie das Redlining auch herunter laden können, wenns wirklich wichtig ist.
